tamr_toolbox/dataset/manage.py

```diff
@@ -121,6 +123,7 @@ def update(
     Raises:
         requests.HTTPError: If any HTTP error is encountered
         ValueError: trying to alter a unified dataset
+        ValueError: Primary keys did not match Tamr dataset primary keys
```

Comment:
awesome!

Reply:
I have since gone back and reverted this change after further discussion during the office hours. My current thought is editing the correct dataset is the user's responsibility and to not add too many safety checks in the code